corehq/util/metrics/__init__.py

```diff
@@ -257,6 +259,12 @@ def new_stop(name=None):
             bucket_tag=bucket_tag, buckets=timing_buckets, bucket_unit='s',
             tags=tags
         )
+        timer_name = ".".join(metric.split('.')[1:])  # remove the 'commcare.' prefix
```

Comment:
do all of our metrics definitely namespace like this? It seems potentially brittle, and the extra word probably doesnt impede readability in sentry too badly.

Reply:
yes they do, it's enforced for all metric names

Reply:
turns out it's only a soft assert: https://github.com/dimagi/commcare-hq/pull/29554